main.py

```diff
@@ -28,7 +28,8 @@ def main():
                 case 'show-birthday':
                     print(contacts.show_birthday(args))
                 case 'birthdays':
```

Comment:
@NotJod563 please add to help command that `birthdays` supports optional parameter - days

Reply:
Done in 3rd commit